src/classes/PMT_Payment_TEST.cls

```diff
@@ -257,6 +257,188 @@ private class PMT_Payment_TEST {
         System.assertEquals(closedWonStage, opp.StageName);
     }
 
+    /**
+     * @description Validate that the accounting rules are enforced properly. If there is a paid payment, it must have a payment date.
+     * If it is an unpaid payment, it must have a scheduled date. 
+     * down the Payment records.
+     */
+    static testMethod void test_EnforceAccountingDataConsistency() {
```

Comment:
in NPSP especially, they want to use 

 @isTest ...

rather than testMethod

Reply:
Updated.